internal/eval_hub/metrics/metrics_test.go

```
@@ -0,0 +1,60 @@
+	if err := metrics.Init(); err != nil {
+		t.Fatalf("metrics.Init: %v", err)
+	}
+
+	ctx := context.Background()
+	metrics.RecordEvaluationJobCreated(ctx, "kubernetes")
+	metrics.RecordEvaluationJobCancelled(ctx)
+	metrics.RecordEvaluationJobRuntimeStartFailed(ctx, "local")
+	metrics.RecordEvaluationJobTerminalState(ctx, api.OverallStateRunning, api.OverallStateCompleted)
+	metrics.RecordBenchmarkRuntimeError(ctx, "kubernetes")
+	metrics.RecordHTTPServerRequest(ctx, http.MethodGet, "/api/v1/health", http.StatusOK)
+	req, _ := http.NewRequest(http.MethodGet, "/api/v1/health", nil)
```

golangci-lint (2.12.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)